### PR TITLE
Removes an event that no longer actually does anything.

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Removes the now depreciated High-Priority Bounty event, since it doesn't do anything.

## Why It's Good For The Game

I hate seeing it roll when it does nothing and it should have been removed in https://github.com/tgstation/tgstation/pull/54029

## Changelog
:cl:
del: Removes the now depreciated high-priority cargo bounty event, that does nothing.
/:cl:

### CODE REVIEW

**Comment by gryffinp** [comment]
F for cargo

**Comment by Ghommie** [comment]
This should be considered a fix rather than a removal for the sake of the feature freeze.

**Comment by necromanceranne** [comment]
This is currently rolling and doing nothing, and the feature it is attached too no longer exists. Iunno this just seems like cleanup to me.

**Comment by Timberpoes** [comment]
We will be accepting fixes and code refractors only.

This is a fix for an oversight and thus is fine.

File: code/modules/events/high_priority_bounty.dm [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-	priority_announce("Central Command has issued a high-priority cargo bounty. Details have been sent to all bounty consoles.", "Nanotrasen Bounty Program")
-
-/datum/round_event/high_priority_bounty/start()
-	var/datum/bounty/B
-	for(var/attempts = 0; attempts < 50; ++attempts)
-		B = random_bounty()
-		if(!B)
-			continue
-		B.mark_high_priority(3)
-		if(try_add_bounty(B))
```
Comment (Jared-Fogle) [inline code/modules/events/high_priority_bounty.dm:18]: I'm confused on why this was able to compile in the first place. When bounties were removed, these procs (and /datum/bounty) should've gone with them as well, no? CC @ArcaneMusic on if this code is dead or not.

Comment (ArcaneMusic) [reply code/modules/events/high_priority_bounty.dm:18]: Well the global bounty list still exists, so it was adding it to basically an invisible list.
List is still defined in `code\modules\cargo\bounty.dm`, and could probably be deleted as it's currently going unused.

Comment (necromanceranne) [reply code/modules/events/high_priority_bounty.dm:18]: Ah well I'll clean that up too once I'm not busy irl.

**Comment by Jared-Fogle** [comment]
Would like to see the dead bounty code get cleaned up, but this is a good start.

**Pull request merged**